src/main/java/com/google/cloud/tools/gradle/appengine/AppEnginePlugin.java

```diff
@@ -54,7 +54,7 @@ private boolean isAppEngineStandard(Project project) {
     }
     // convention based lookup of appengine-web.xml as a fallback
     Path appengineWebXml =
-        project.getProjectDir().toPath().resolve("src/main/java/webapp/WEB-INF/appengine-web.xml");
+        project.getProjectDir().toPath().resolve("src/main/webapp/WEB-INF/appengine-web.xml");
```

Comment:
Yeah we do that, see [line 48](https://github.com/GoogleCloudPlatform/app-gradle-plugin/pull/139/files/5e7ebc6d2d2b781aaa877404222f0f5c4865310a#diff-e63a80ab373530959c0f994705e2a85cR48), this just covers the case that the war plugin is applied *after* the appengine plugin.